dist/index.js

```diff
@@ -16001,7 +16001,7 @@ const setDynamicVars = () => {
 		context.ACTOR = github.context.actor
 		context.REF = github.context.ref
 		context.SHA = github.context.sha
-		context.BRANCH = github.context.ref.substr(11)
+		context.BRANCH = github.context.ref.substring(11)
```

Comment:
Maybe one could even be more explicit and state where the `11` char length comes from?
```suggestion
		context.BRANCH = github.context.ref.substring("refs/heads/".length)
```